README.md

```diff
@@ -160,66 +173,26 @@ Or run a single test, like so:
 docker-compose -f docker-compose.yml -f tests/docker-compose.yml run --rm app pytest tests/test_person.py::test_no_existing_sources
 ```
 
-## Using the Google Sheet import script
+## Using the Google Drive import script
 
-If you're reading this and you're asking yourself, "Why on earth would I ever
-want to import data into this system from a Google Sheet? What a nightmare!"
-well, greetings from the past where that was a regular feature of our routine.
-However, it was not as bad as you might think thanks to the brilliant work of
-the SFM team.
-
-The script to import data from a Google Sheet exists as a management command
+The script to import data from Google Drive exists as a management command
 within the `sfm_pc` directory of this project predictably called
-`import_google_doc`. Before actually running the script, you'll need to take
-a few steps.
-
-The google sheets need to follow this format: https://docs.google.com/spreadsheets/d/12Kap4YtzWPtCdJy-iknF1aDaoIDMij4GU8E2-HlZkAo/edit#gid=597380077
+`import_google_doc`. As of February 2021, spreadsheets to be imported must follow
+this format: https://docs.google.com/spreadsheets/d/1-U_pVNDWFIlG8jLZLVujmV_FZ0H4IsbtfCeRiM8HeNQ/edit?usp=sharing
 
-### Import OpenStreetMap data
-
-As the Google Sheet importer is importing, it is creating relationships between
-the various entities within the data and OSM nodes, ways and relations. In
-order to speed that process up, we bulk download the data and insert it into
-the location data table directly (rather than making API calls for every lookup
-or some such thing). To do that, update your settings file so that it includes
-the countries you'd like to import. You should see a setting called `OSM_DATA`
-that includes a list of dictionaries that look like this:
-
-```
-{
-    'country': 'Mexico',
-    'pbf_url': 'http://download.geofabrik.de/north-america/mexico-latest.osm.pbf',
-    'relation_id': '114686',
-    'country_code': 'mx',
-}
-```
-
-* `country` is the nice name for the country
-* `pbf_url` is a location on the internet where you can download a PBF snapshot
-  of the OSM data for that country (we've been using Geofabrik and it's been
-  working just fine)
-* `relation_id` is the OSM id for the relation that represents the country's
-  boundary
-* `country_code` is the two letter ISO country code for the country
-
-Once you have that sorted out, you'll need to make sure you have `osm2pgsql`
-installed (this is the command line tool we use under the hood to turn the PBF
-file into a PostgreSQL table). Once that is installed, you should be able to
-run the `import_osm` management command:
-
-```
-python manage.py import_osm
-```
+### Import data from Google Drive
```

Comment:
I updated this entire section based on the revised Makefile and import docket workflow we've worked out with SFM, so it has been tested handily over the last week.